### ISSUE TITLE
PayloadMismatchError in v3.3.3 and v3.4.0 when making v1 POST call with query parameters in the URI

### ISSUE THREAD

**Issue by jeffshurtliff** [jeffshurtliff/khoros#27]
**Description**
When passing query parameters in the URI for a Community API v1 POST call, the following exception exception is raised.

```python
khoros.errors.exceptions.PayloadMismatchError: More than one payload was provided for the POST request when only one is permitted.
```

**Impacted Version**
- 3.3.3
- 3.4.0

**Examples**
```python
update_setting = khoros.v1.post('/users/id/46/settings/name/custom.custom_settings/set?value=testing', return_json=False, params_in_uri=True)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Users\redacted\Development\khoros\khoros\core.py", line 802, in post
    params_in_uri=params_in_uri, json_payload=json_payload)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 756, in make_v1_request
    headers=header)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 459, in post_request_with_retries
    multipart=multipart, content_type=content_type.lower())
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 400, in payload_request_with_retries
    raise errors.exceptions.PayloadMismatchError(request_type=request_type.upper())
khoros.errors.exceptions.PayloadMismatchError: More than one payload was provided for the POST request when only one is permitted.
update_setting = khoros.v1.post('/users/id/46/settings/name/custom.custom_settings/set?value=testing', return_json=False, params_in_uri=False)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Users\redacted\Development\khoros\khoros\core.py", line 802, in post
    params_in_uri=params_in_uri, json_payload=json_payload)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 763, in make_v1_request
    khoros_object=khoros_object, headers=header)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 459, in post_request_with_retries
    multipart=multipart, content_type=content_type.lower())
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 400, in payload_request_with_retries
    raise errors.exceptions.PayloadMismatchError(request_type=request_type.upper())
khoros.errors.exceptions.PayloadMismatchError: More than one payload was provided for the POST request when only one is permitted.
update_setting = khoros.v1.post('/users/id/46/settings/name/custom.custom_settings/set?value=testing', return_json=False)
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Users\redacted\Development\khoros\khoros\core.py", line 802, in post
    params_in_uri=params_in_uri, json_payload=json_payload)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 763, in make_v1_request
    khoros_object=khoros_object, headers=header)
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 459, in post_request_with_retries
    multipart=multipart, content_type=content_type.lower())
  File "C:\Users\redacted\Development\khoros\khoros\api.py", line 400, in payload_request_with_retries
    raise errors.exceptions.PayloadMismatchError(request_type=request_type.upper())
khoros.errors.exceptions.PayloadMismatchError: More than one payload was provided for the POST request when only one is permitted.
```

**Comment by jeffshurtliff** [comment]
Issue will be resolved in v3.5.0.

**Comment by jeffshurtliff** [comment]
Issue is resolved in v3.5.0.
